src/xdp-portal-impl.c

```diff
@@ -269,16 +269,11 @@ sort_impl_by_use_in_and_name (gconstpointer a,
 }
 
 void
-load_installed_portals (gboolean opt_verbose)
+load_installed_portals_dir (gboolean    opt_verbose,
+                        const char *portal_dir)
```

Comment:
Something has gone wrong with the indentation here, are you using tabs? (If yes: please don't)